src/ida_lunde_naalsund_ex/ex04/myrand.py

```diff
@@ -0,0 +1,103 @@
+# -*- coding: utf-8 -*-
+
+__author__ = "Ida Lunde Naalsund"
+__email__ = "idna@nmbu.no"
+
+
+class LCGRand:
+    """
+    Returns random numbers by implementing the linear congruential
+    generator function.
+
+    Attributes
+    ----------
+    a : int
+        Constant given
+
+    m : int
+        Constant given
+
+    """
```

Comment:
As neither a or m are useful for the user as attributes, (they are only arguments to the constructor) they don't need to be documented. Essential documentation for a class should be given in the __init__.